conda/gateways/disk/create.py

```diff
@@ -36,6 +36,14 @@
     exit(%(func)s())
 """)
 
+private_pkg_entry_point_template = dals("""
+import os
+import sys
+if __name__ == '__main__':
+    exec_path = os.path.join("%(private_env_path)s", "bin", "%(app_name)s")
```

Comment:
What about env `activate` here.  Are we punting on that for now?  Or maybe you've got it covered later and I haven't seen it yet.